gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1691,7 +1691,7 @@ MM_ParallelGlobalGC::reportGCStart(MM_EnvironmentBase *env)
 	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
 
 #if defined(OMR_GC_MODRON_SCAVENGER)
-	scavengerCount = _extensions->scavengerStats._gcCount;
+	scavengerCount = _extensions->incrementScavengerStats._gcCount;
```

Comment:
This one is somewhere in-between pure cycle and pure increment, but resembles more to increment - I can see that reportGCStart is always called when reportGlobalGCIncrementStart is called, but reportGCCycle is sometimes handled separately.

So, the change is correct (my comment is just there to justify it).